src/OpenTelemetry.Instrumentation.AWSLambda/.publicApi/netstandard2.0/PublicAPI.Shipped.txt

```diff
@@ -0,0 +1 @@
+#nullable enable
```

Comment:
Why is this in the Shipped API? I think this is wrong.

Reply:
It was added automatically after suggestion.

Reply:
There was some warnings when it was removed.

Reply:
Have you tried adding it to the Unshipped.txt instead? That's were it belongs I think.

Reply:
Yes, moving Unshipped helps.